src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -1163,6 +1165,42 @@ export default class SourceFileLinter {
 		}
 	}
 
+	#analyzeFragmentLoad(node: ts.CallExpression) {
+		if (!node.arguments?.length) {
+			return;
+		}
+
+		const optionsArg = node.arguments[0];
+
+		if (!ts.isObjectLiteralExpression(optionsArg)) {
+			return;
+		}
+
+		// Find "type" property
+		let typeProperty;
+		for (const property of optionsArg.properties) {
+			if (!ts.isPropertyAssignment(property)) {
+				continue;
+			}
+			if (
+				(ts.isIdentifier(property.name) || ts.isStringLiteral(property.name)) &&
+				property.name.text === "type"
+			) {
+				typeProperty = property;
+				break;
+			}
+		}
+
+		if (typeProperty && ts.isStringLiteralLike(typeProperty.initializer)) {
+			const typeValue = typeProperty.initializer.text;
+			if (typeValue === "HTML") {
+				this.#reporter.addMessage(MESSAGE.PARTIALLY_DEPRECATED_FRAGMENT_LOAD, {
+					typeValue,
+				}, node);
+			}
+		}
+	}
```

Comment:
This code is 99% like the `#analyzeViewCreate`. 
Is it possible to consolidate those and just do the check & message separately

Reply:
Maybe, we need to somehow extract and reuse the type field extraction. I have it also in my change:
https://github.com/SAP/ui5-linter/pull/480/files#diff-7bc0071c1214325807d3a82a222b097360e3b0020aae68d0575dbae4aa43b99aR1151

Reply:
Yeah, some better reuse would be great.